pages/api/graphql.ts

```diff
@@ -16,7 +17,25 @@ const handler: any = nextConnect() // For session middleware. TODO: Need to defi
 const apolloServer = new ApolloServer({
   typeDefs,
   resolvers,
-  context: ({ req }) => ({ req }) // This lets GraphQL have access to sessions
+  context: async ({ req }) => {
```

Comment:
This should not be done in context. We don't want this query to run everytime a request is made to GraphQL. In addition, having `Promise.all` iseems really inefficient when we could just include all `Submission` and `UserLesson` in one query.